Updated READMEs (#336)
twilio/voice-quickstart-android: README.md

Get started with Voice on Android:

- [Quickstart](#quickstart) - Run the quickstart app

> - [Examples](#example) - Customize your voice experience with custom audio device example 

Reviewer: ```suggestion
- [Examples](#example) - Customize your voice experience with these examples
```

```

### Configuring Hardware Audio Effects

> #### Voice Android SDK Version 5.2.x

Reviewer: ```suggestion
#### Voice Android SDK Version 5.2.x+
```

Our library performs acoustic echo cancellation (AEC) and noise suppression (NS) using device hardware by default. Using device hardware is more efficient, but some devices do not implement these audio effects well. If you are experiencing echo or background noise on certain devices reference the following snippet for enabling software implementations of AEC and NS.

    /*
     * Execute any time before invoking `Voice.connect(...)` or `CallInvite.accept(...)`.
     */

    // Use software AEC
    DefaultAudioDevice defaultAudioDevice = new DefaultAudioDevice();
    defaultAudioDevice.setUseHardwareAcousticEchoCanceler(false);
    Voice.setAudioDevice(defaultAudioDevice);

    // Use sofware NS
    DefaultAudioDevice defaultAudioDevice = new DefaultAudioDevice();
    defaultAudioDevice.setUseHardwareNoiseSuppressor(false);

>     Voice.setAudioDevice(defaultAudioDevice);
> #### Voice Android SDK Version below 5.1.x

Reviewer: ```suggestion
    Voice.setAudioDevice(defaultAudioDevice);
    
    
#### Voice Android SDK Version below 5.1.x
```
